devicesimulation/single-vm/scripts/update.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -ex
+
+# Where to download scripts from
+REPOSITORY="https://raw.githubusercontent.com/Azure/pcs-cli/1.0.0-preview.3/remotemonitoring/single-vm"
```

Comment:
use 'https://raw.githubusercontent.com/Azure/pcs-cli/azure-iot-pcs-simulation/devicesimulation/single-vm'?

Reply:
Currently, the suggested branch is not there. I'll suggest to merge this PR first, then change the branch in a different PR. So we will not break the source branch

Reply:
it is here: https://github.com/Azure/pcs-cli/tree/azure-iot-pcs-simulation

Reply:
let's fix the values now, one fewer TODO to track

Reply:
Fixed